### ISSUE TITLE
Branches API call does not expose ID number

### ISSUE THREAD

**Issue by martin9700** [AtlassianPS/BitbucketPS#2]
Not sure if this is the right place to mention this, but!  When calling the

$server/rest/api/1.0/projects/$ProjectKey/repos/$Repo/branches

API call, it does not expose the branch ID number, making it impossible to correlate commits to the branch.  It looks like commits does expose the "parent ID" and I'm assuming this would be the ID number of the branch.

Alternative would be to add a branch search to the commits API call.

**Comment by lipkau** [comment]
Good catch. Did you fix this in your PR?

**Comment by martin9700** [comment]
No, API doesn’t publish the number so nothing we can do downstream.

**Comment by lipkau** [comment]
I found this API that can list all commits of a specific branch:
https://docs.atlassian.com/bitbucket-server/rest/5.6.2/bitbucket-rest.html#idm46036344406192
